tests/ui/test_ui_tools.py

```diff
@@ -635,6 +635,39 @@ def test_keypress_GO_BACK(self, mocker, topic_view, key):
         topic_view.set_focus.assert_called_once_with("body")
         assert topic_view.log == self.topics_btn_list
 
+    def test_mouse_event(self, mocker, topic_view):
+        mocker.patch.object(topic_view, 'keypress')
+        size = (200, 20)
+        col = 1
+        row = 1
+        focus = "WIDGET"
+
+        # Scroll wheel up: 4.
+        topic_view.mouse_event(size, "mouse press", 4, col, row, focus)
+        topic_view.keypress.assert_called_once_with(
+            size, next(iter(keys_for_command('GO_UP')))
+        )
+
+        # Scroll wheel down: 5.
+        topic_view.mouse_event(size, "mouse press", 5, col, row, focus)
+        topic_view.keypress.assert_called_with(
+            size, next(iter(keys_for_command('GO_DOWN')))
+        )
```

Comment:
Same response here as for the other two-part test you altered, except you're adding it from scratch here, so there's no need to refactor.